lib/Cleantalk/Antispam/Integrations/LearnPress.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Cleantalk\Antispam\Integrations;
+
+class LearnPress extends IntegrationBase
+{
+    public function getDataForChecking($argument)
+    {
+        if ( ! empty($_POST) ) {
+            return ct_gfa(apply_filters('apbct__filter_post', $_POST));
```

Comment:
Need to pass `$data['register'] = true;` flag to the returned array.

Reply:
fixed https://github.com/CleanTalk/wordpress-antispam/pull/385/commits/d7d5e7027d4ea2f09690eded5c41895f212a3d23